### DISCUSSION TITLE
2024/06/20/Transformer/

### DISCUSSION THREAD

**Discussion by giscus** [AbyssSkb/abyssskb.github.io#3] [Announcements]
# 2024/06/20/Transformer/

前言 高二下学期，一次偶然的机会，我接触到了 AI。当时的我加入到了一位好朋友的行列中，一起学习机器学习。机器学习（Machine Learning）是一门人工智能的科学，该领域的主要研究对象是人工智能，特别是如何在经验学习中改善具体算法的性能。 在那段快乐的时光，我接触到了许多机器学习领域的知识。在听完吴恩达的机器学习课程 后，我开始接触深度学习。深度学习（Deep Learning）是机器学习

https://abyssskb.github.io/2024/06/20/Transformer/

**Comment by AbyssSkb** [comment]
泪目，还是想哭😭
